resolwe_docker_images/rnaseq/packages-manual/arriba.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+. /var/cache/build/packages-manual/common.sh
+
+download_and_verify \
+    "suhrig" \
+    "arriba" \
+    "2.4.0" \
+    "a5173f44195d7f864aab95972d0cc3da85671c4b7e602e5a4e1a4fc143810e4a" \
+    "https://github.com/suhrig/arriba/releases/download/v2.4.0/arriba_v2.4.0.tar.gz" \
+    "arriba_v2.4.0" \
+    "arriba_v2.4.0.tar.gz"
+
+make
```

Comment:
Why building this from source? Arriba executable should be part of the downloaded package. Have you tried using that?

Also, the downloaded package is quite large. Unecessary files should be deleted, especially the reference fusion files fasta files, as those will not be stored in the Docker image and are also the largest.